### ISSUE TITLE
Pipeline: prefixed accessor warning, overlap

### ISSUE THREAD

**Issue by dtzSiFive** [llvm/circt#3900]
Building 0068dfd7e297a3f74cb0ab95af6cc0a3dad15894 produces the following:

```
/home/will/src/sifive/circt/include/circt/Dialect/Pipeline/Pipeline.td:193:1: note: Skipping generation of prefixed accessor `getOperands` as it overlaps with default one; generating raw form (`operands`) still
def ReturnOp : Op<Pipeline_Dialect, "return", [Terminator]> {
```

cc #3889 .

**Comment by mikeurbach** [comment]
I can take a look at this one.

**Comment by dtzSiFive** [comment]
Great! Hopefully not much trouble, just wanted to flag it.  Thanks!

**Comment by mikeurbach** [comment]
Closed by 54a2b86d1953d20ac3556444ded9e5cd176c7a7f

**Comment by dtzSiFive** [comment]
Awesome, thanks!
